doc/nrf/config_and_build/config_and_build_system.rst

```diff
@@ -255,6 +258,16 @@ For example, when building a sample that enables :kconfig:option:`CONFIG_BT_EXT_
 
 To disable these warnings, disable the :kconfig:option:`CONFIG_WARN_EXPERIMENTAL` Kconfig option.
 
+.. _sysbuild_enabled_ncs:
+
+Sysbuild enabled by default
+===========================
+
+In the |NCS|, building an application using the :ref:`standard building procedure <building>` automatically includes :ref:`configuration_system_overview_sysbuild` (the ``--sysbuild`` parameter).
+For this reason, unlike in Zephyr, ``--sysbuild`` does not have to be explicitly mentioned in the command prompt.
```

Comment:
This should make it clear it is NCS repos, not NCS in general, if a customer builds their own manifest then it will not enable sysbuild by default in their custom included repos but will in the ncs repos inside of that manifest